src/vm/loaderallocator.hpp

```diff
@@ -485,7 +485,6 @@ typedef VPTR(LoaderAllocator) PTR_LoaderAllocator;
 class GlobalLoaderAllocator : public LoaderAllocator
 {
     VPTR_VTABLE_CLASS(GlobalLoaderAllocator, LoaderAllocator)
-    VPTR_UNIQUE_OVERRIDE(VPTRU_LoaderAllocator+1)
```

Comment:
> I see, yea having virtual instead of override would be fine I think, at least it would do the expected thing

Note that currently it does the expecting thing (not tagging `virtual` for a base method marked as `virtual` will still make it `virtual`... quite a few places in CoreCLR that didn't bother...). But at least re-tagging with `virtual` will make it a bit more obvious (also this [issue on stackoverflow](http://stackoverflow.com/questions/4895294/c-virtual-keyword-for-functions-in-derived-classes-is-it-necessary) says that some compiler can emit warnings so...)